cli/pkg/fileutil/file.go

```diff
@@ -0,0 +1,99 @@
+package fileutil
```

Comment:
This new `fileutil` package is based from the old `file` package, but it's more limited in scope. It only focuses on handling the ID injection on applied files